tests/many_to_many/tests.py

```diff
@@ -332,6 +332,45 @@ def test_remove(self):
             ])
         self.assertQuerysetEqual(self.a3.publications.all(), [])
 
+    def test_set(self):
+        self.p2.article_set.set(self.a4, self.a3)
+        self.assertQuerysetEqual(self.p2.article_set.all(),
+            [
```

Comment:
I'd prefer if you kept the same indent style as the above tests (but I guess you've copied the style of the surrounding code)